src/libs/actions/Search.ts

```diff
@@ -2,9 +2,9 @@ import * as API from '@libs/API';
 import {READ_COMMANDS} from '@libs/API/types';
 import * as SearchUtils from '@libs/SearchUtils';
 
-function search(query: string) {
-    const hash = SearchUtils.getQueryHash(query);
-    API.read(READ_COMMANDS.SEARCH, {query, hash});
+function search(query: string, policyID?: string) {
+    const hash = SearchUtils.getQueryHash(query, policyID);
```

Comment:
I suggest we only compute `hash` in the search component and pass it here along with the other arguments.
Feels unnecessary to count hash twice, and it will make this function super simple.